indico_jacow/controllers.py

```diff
@@ -119,19 +119,18 @@ def _process(self):
 
 
 def _append_affiliation_data_fields(headers, rows, items):
+    def make_address(affiliation):
+        address = ' '.join(filter(None, (affiliation.postcode, affiliation.city)))
+        return ', '.join(filter(None, (affiliation.street, address)))
+
     def full_name_and_data(person, data):
-        return f'{person.full_name} ({data})' if data else person.full_name
+        return f"{person.full_name} ({'; '.join(data)})" if data else person.full_name
```

Comment:
I'd rather use those double quotes inside the string instead of outside.

Reply:
changing it to `f'{person.full_name} ({"; ".join(data)})'` was making the linting fail (idk if thats intended)... moved the `'; '.join(data)` outside instead

Reply:
IIRC flake8 on Python 3.12 wants you to use the same quotes inside and outside.
Splitting it into separate statements to avoid nesting quotes is indeed the best option!